.github/workflows/svg-compare-all-views.yml

```diff
@@ -0,0 +1,88 @@
+            - uses: ./.github/actions/build-tsc
+
+            - name: Set branch name and token for OWID runs
+              if: ${{ github.repository_owner == 'owid' }}
+              shell: bash
+              run: |
+                  echo "PUSH_BRANCH_NAME=${{ github.ref_name }}" >> $GITHUB_ENV
+                  echo "TOKEN=${{ secrets.GITHUBPAT }}" >> $GITHUB_ENV
+
+            - name: Set branch name and token for runs from a fork
+              if: ${{ github.repository_owner != 'owid' }}
+              shell: bash
+              run: |
+                  echo "PUSH_BRANCH_NAME=${{ github.repository_owner }}/${{ github.ref_name }}" >> $GITHUB_ENV
+                  echo "TOKEN=${{ github.token }}" >> $GITHUB_ENV
+
+            # Checkout the owid-grapher-svgs repo into a subfolder. Use a Personal Access Token for checkout so the
+            # action will have permission to push later on if required.
+            - name: Clone svg tester repo
+              uses: actions/checkout@v3
+              with:
+                  repository: "owid/owid-grapher-svgs"
+                  path: owid-grapher-svgs
+                  token: ${{ env.TOKEN }}
+
+            # Create a branch on that repo that matches the branch name we are on in the owid-grapher repo
+            # but only do this if we are not on master in owid-graphers (in this case we want to commit and push on master
+            # in owid-grapher-svgs as well)
+            - name: create owid-grapher-svgs local branch
+              if: ${{ env.PUSH_BRANCH_NAME != 'master'}}
+              working-directory: owid-grapher-svgs
+              continue-on-error: true
+              run: git branch ${{ env.PUSH_BRANCH_NAME }} || echo "Branch ${{ env.PUSH_BRANCH_NAME }} already exists"
+
+            # Checkout the branch we just created (but not if we are on master in owid-grapher)
+            - name: checkout owid-grapher-svgs local branch
+              if: ${{ env.PUSH_BRANCH_NAME != 'master'}}
+              working-directory: owid-grapher-svgs
+              run: git checkout ${{ env.PUSH_BRANCH_NAME }}
+
+            # Run the verify tool overwriting any svgs. Stdout is piped to compare-result which will be a 0 byte file if everything works or contain failed grapher ids otherwise
+            - name: Generate SVGs and compare to reference
+              id: run-verify-graphs
+              continue-on-error: true
+              env:
+                  # using "ternary operator" from https://github.com/actions/runner/issues/409#issuecomment-752775072
+                  RM_ON_ERROR: ${{ env.PUSH_BRANCH_NAME == 'master' && '' || '--rm-on-error' }}
+              run: node --enable-source-maps itsJustJavascript/devTools/svgTester/verify-graphs.js -i owid-grapher-svgs/configs -o owid-grapher-svgs/all-views/svg -r owid-grapher-svgs/all-views/svg --ids-from-file owid-grapher-svgs/most-viewed-charts.txt --all-views $RM_ON_ERROR > compare-result
```

Comment:
The workflow is identical to the current one. This is the only line of code that meaningfully changed.